src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -127,6 +128,8 @@ private static String execute(File workingDirectory, String command, Map<String,
                 public String apply(String s, String s2) {
                     if (s2.startsWith("---")) {
                         notificationFound = true;
+                    } else if (s2.startsWith("W")) { // workaround for ingress warning in json
```

Comment:
Seems like an ugly patch would be to read stdout and stderr separately instead of aggregation

Reply:
this implies a modification in intellij-common.

Reply:
Should be there as @evidolob is using it already.
See https://github.com/redhat-developer/intellij-common/blob/main/src/main/java/com/redhat/devtools/intellij/common/utils/ExecHelper.java#L185

Reply:
correct. it's there from quite some time....